### PR TITLE
4 lint and slight refactor, updates to work with new birdnet output format

### PR DESCRIPTION
**Pull request by kgarwoodsdzwa**
This began as a lint pass, but I realized that birdnet had adjusted their output format from when these scripts were created to handle the previous format. They now include an extra class 'nocall' even when filtering for burrowing owl only calls. These needed to be parsed out and dealt with in addition to the linting. Arg parsing was added to all the scripts to make then easier to run from the command line. There are still a few spots where the user needs to adjust some numbers based on their example, but the current numbers are set for a specific example. Both of the normalization scripts require this user input, and are based on the length of your .wav file. Future revisions will obtain the length of the file using tools such as librosa so that the user need not change the code.

### CODE REVIEW

File: assess_birdnet/aggregate_audio_analysis.py [hunk 1]
```diff
@@ -1,8 +1,29 @@
+"""Aggregate Birdnet Output Files
```
Comment (iingram) [inline assess_birdnet/aggregate_audio_analysis.py:1]: To be strict to Google docstrings first lines should end with a period.

There are a few places in these 4 files where this comes up but a) I am only going to note it here and b) I am going to approve the PR anyway because we can fix these little things on the next effort on these tools.

File: assess_birdnet/aggregate_audio_analysis.py [hunk 2]
```diff
@@ -14,35 +35,53 @@ def main(root_dir):
...
+                "High Freq (Hz)", "Species Code",
+                "Common Name",
+                "Confidence",
+                "File Name",
+                "Extra 1",
+                "Extra 2"
             ]
 
-            with open(output_csv_file, mode='w', newline='') as csvfile:
+            with open(
```
Comment (iingram) [inline assess_birdnet/aggregate_audio_analysis.py:55]: I do feel you might format this a little differently as we discussed but no biggie.

File: assess_birdnet/assess_performance.py [hunk 3]
```diff
@@ -1,23 +1,58 @@
...
+"""
+import argparse
 import pandas as pd
-from sklearn.metrics import confusion_matrix, accuracy_score, precision_score, recall_score, f1_score
+from sklearn.metrics import confusion_matrix, accuracy_score, precision_score
+from sklearn.metrics import recall_score, f1_score
+
+
+def main(human_labeled, birdnet_labeled):
+    """Evaluate Birdnet metrics
```
Comment (iingram) [inline assess_birdnet/assess_performance.py:19]: To fit Google docstring convention you need a blank line after this summary line.  

I think this comes up a couple of places in these 4 files but going to only note it here and also approve the PR

File: assess_birdnet/assess_performance.py [hunk 4]
```diff
@@ -1,23 +1,58 @@
...
+    y_true = scored_data['Label'].map({'yes': 1, 'no': 0}).values
+    y_pred = ml_output['Label'].map({'yes': 1, 'no': 0}).values
 
-y_true = scored_data['Label'].map({'yes': 1, 'no': 0}).values
-y_pred = ml_output['Label'].map({'yes': 1, 'no': 0}).values
+    confusion_m = confusion_matrix(y_true, y_pred)
+    accuracy = accuracy_score(y_true, y_pred)
+    precision = precision_score(y_true, y_pred)
+    recall = recall_score(y_true, y_pred)
+    f_1 = f1_score(y_true, y_pred)
```
Comment (iingram) [inline assess_birdnet/assess_performance.py:37]: You might just call `f_1` `f1_score`

File: assess_birdnet/normalize_scored_output.py [hunk 5]
```diff
@@ -1,33 +1,85 @@
...
-    end_time = start_time + row['DURATION']
+
+def mark_intervals(row, chunks_df):
+    """Labeling detections
+    Function to relabel the row in the dataframe
+    to yes if the human labels marked a vocalization
+    at that point.
+
+    Args:
+        row (pandas.Series object): The current row in the human labeled
```
Comment (iingram) [inline assess_birdnet/normalize_scored_output.py:60]: You don't need to write "object" The fact that this is an object is called out by the capitalization of the class type the object is an instance of.  Ditto with `pandas.Dataframe` below

File: assess_birdnet/normalize_scored_output.py [hunk 6]
```diff
@@ -1,33 +1,85 @@
...
+        data that matches the audio file of interest.
```
Comment (iingram) [inline assess_birdnet/normalize_scored_output.py:61]: The continuation line here should be indented an additional 4 spaces.

File: assess_birdnet/normalize_scored_output.py [hunk 7]
```diff
@@ -1,33 +1,85 @@
...
-chunks_df = pd.DataFrame(chunks_data)
+    chunks_df.to_csv(adjusted_labels, index=False)
+    print(f"File {adjusted_labels} created successfully.")
 
-def mark_intervals(row):
-    start_time = row['OFFSET']
-    end_time = start_time + row['DURATION']
+
+def mark_intervals(row, chunks_df):
+    """Labeling detections
```
Comment (iingram) [inline assess_birdnet/normalize_scored_output.py:54]: Use the more direct verb for functions i.e Relabel detections ...and maybe also could tuck in a little more detail about how e.g. "Relabel detections using human labels" or something like that

Comment (iingram) [reply assess_birdnet/normalize_scored_output.py:54]: There's another place or two where the same note could be applied to a function docstring.

File: assess_birdnet/normalize_scored_output.py (hunk 5 above)
Comment (iingram) [reply assess_birdnet/normalize_scored_output.py:60]: Saw another place in the code with same thing.

**Review by iingram — APPROVED** [review]
Looks good.  I did have some comments but they are things you can address in the next set of things you are planning to do on these four scripts so I am going to approve and merge.

I also saw a few long line warnings out of flake8. I feel the same way about those.  I didn't actually check which lines they were.

And, functionally, the scripts performed as advertised using the example data and instructions you gave me.
